.github/workflows/update_modules.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: python ./_repo_generator.py
       - name: Commit packages
         run: |
-          git add *.zip
+          git add '*.zip'
```

Comment:
If this doesn't work, the [documentation](https://git-scm.com/docs/git-add#_examples) suggests `git add \*.zip`.

Reply:
I think it is meant to be `git add ./\*.zip`

Reply:
@jurialmunkey Would that one limit the search to subdirectories only?

Reply:
`./` is the current directory
`\*` escapes the wildcard to prevent the shell from expanding it

Reply:
Really though I would expect either to work - just if we're following the documentation example to the letter then we should include the current directory explicitly with `./`